docsets/JUCE/docset.json

```diff
@@ -12,7 +12,7 @@
     ],
     "specific_versions": [
         {
-            "version": "7.0.4",
+            "version": "7.0.5",
             "archive": "versions/7.0.4/JUCE.tgz"
```

Comment:
You need to change this path too I think.

Reply:
good catch.